lib/Common/DataStructures/SparseBitVector.h

```diff
@@ -486,6 +508,7 @@ template <class TAllocator>
 BVSparseNode<TAllocator> *
 BVSparse<TAllocator>::DeleteNode(BVSparseNode *node, bool bResetLastUsed)
 {
+    this->tail = nullptr;
```

Comment:
Do we necessarily only delete the last node? If so, shouldn't we update the tail to be the new last node?

Reply:
updated the variable name.

Reply:
Ah, that clears things up a lot, thanks!